components/brave_wallet/browser/swap_response_parser.cc

```diff
@@ -21,6 +25,50 @@ bool ParseResultFromDict(const base::DictionaryValue* response_dict,
   return true;
 }
 
+bool ParseUint64ResultFromValue(const base::Value& value,
```

Comment:
All of these functions should be returning absl::optional.

Reply:
Fixed here e918beffe639e2f65fd97084ce58e05f75722925.

Reply:
By the way, I realised we don't need these dedicated methods, since `Find*Key` methods already return `absl::optional`.